google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITQueryTest.java

```diff
@@ -78,9 +76,13 @@ public void simple() {
 
   @Test
   public void badQuery() {
-    expectedException.expect(isSpannerException(ErrorCode.INVALID_ARGUMENT));
-    expectedException.expectMessage("Unrecognized name: Apples");
-    execute(Statement.of("SELECT Apples AND Oranges"), Type.int64());
+    try {
+      execute(Statement.of("SELECT Apples AND Oranges"), Type.int64());
+      fail("");
```

Comment:
nit: Here and below, add "Expected exception" message.

Reply:
Done